src/App.js

```diff
@@ -1,16 +1,85 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import ColorChoice from './components/ColorChoice';
+import { useState } from 'react';
 
 const App = () => {
+  const [entriesData, setEntries] = useState(chatMessages);
```

Comment:
Style: The naming convention for `useState` is generally `[thing, setThing]`:

```suggestion
  const [entries, setEntries] = useState(chatMessages);
```